src/components/Post.js

```diff
@@ -304,14 +304,14 @@ export class CommentSection extends React.Component {
     this.setState(newState)
   }
 
-  startedTyping () {
+  startedTyping = () => {
```

Comment:
note this syntax which creates a bound function, so we can just refer to `this.startedTyping` elsewhere instead of `this.startedTyping.bind(this)`